packages/e2e-tests/specs/experiments/navigation-editor.test.js

```diff
@@ -679,4 +680,143 @@ describe.skip( 'Navigation editor', () => {
 			expect( lastItemAttributes.isTopLevelLink ).toBeTruthy();
 		} );
 	} );
+	describe( 'Delete menu button', () => {
+		afterEach( async () => {
+			await setBrowserViewport( 'large' );
+		} );
+		it.each( [ 'large', 'small' ] )(
+			`should retain menu when confirmation is canceled and the viewport is %s`,
+			async ( viewport ) => {
+				const menuName = 'Menu delete test';
+				await createMenu( { name: menuName }, menuItemsFixture );
+				await visitNavigationEditor();
+				await setBrowserViewport( viewport );
+				// Wait for the header to show the menu name.
+				await page.waitForXPath(
+					`//h2[contains(text(), "${ menuName }")]`
+				);
+
+				if ( viewport === 'small' ) {
+					const openSettingsSidebar = await page.waitForXPath(
+						'//button[@aria-label="Settings"][@aria-expanded="false"]'
+					);
+					await openSettingsSidebar.click();
+				}
+
+				const deleteMenuButton = await page.waitForXPath(
+					'//button[text()="Delete menu"]'
+				);
```

Comment:
Should we look for this "Delete menu" inside the sidebar (an element with `role="region"` and  `aria-label="Navigation settings"`) ?

Reply:
✅  e7653f3